### PR TITLE
chore: update hilla 2.2.0.alpha10

### PR DESCRIPTION
**Pull request by ZheSun88**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,27 +80,27 @@
     "@vaadin/vaadin-development-mode-detector": "2.0.6",
     "@vaadin/vaadin-lumo-styles": "24.2.0-alpha11",
     "@vaadin/vaadin-material-styles": "24.2.0-alpha11",
-    "@vaadin/vaadin-themable-mixin": "24.2.0-alpha11",
     "@vaadin/vaadin-usage-statistics": "2.1.2",
     "@vaadin/vertical-layout": "24.2.0-alpha11",
     "@vaadin/virtual-list": "24.2.0-alpha11",
     "construct-style-sheets-polyfill": "3.1.0",
-    "lit": "2.8.0",
+    "lit": "2.7.6",
```
Comment (Artur-) [inline package.json:87]: no.. why?

Comment (Artur-) [reply package.json:87]: Is this because Flow was downgraded?

Comment (ZheSun88) [reply package.json:87]: it seems 
we need to pick this one to flow 24.1 https://github.com/vaadin/flow/pull/17414

Comment (Artur-) [reply package.json:87]: Let's not change how we build Flow just because Hilla downgraded the dependency

**Review by Artur- — APPROVED** [review]

**Pull request merged**